src/app/teams/teams-view.component.html

```diff
@@ -172,8 +175,11 @@ <h3 *ngIf="mode==='services'" class="margin-lr-3 ellipsis-title">{{configuration
               <mat-card *ngFor="let resource of resources" (click)="resource.resource._attachments ? openResource(resource.resource._id) : false" i18n-matTooltip matTooltip="There is no content for this resource" [matTooltipDisabled]="resource.resource._attachments" [ngClass]="{'cursor-pointer': resource.resource._attachments}">
                 <div class="mat-subheading-2">
                   <span *ngIf="resource.resource!==undefined">
-                    {{resource.resource.title}}
+                    <b>{{resource.resource.title}}</b>
                   </span>
+                  <mat-card-content>
```

Comment:
Please move the `mat-card-content` element outside of the `div` element.

Reply:
Thanks for your comment @paulbert